code/modules/mob/living/simple_animal/hostile/thing.dm

```diff
@@ -0,0 +1,576 @@
+/mob/living/simple_animal/hostile/thing  // Originally named "thing".
+	name = "unknown"
+	desc = "What is that?"
+	icon = 'icons/mob/mob.dmi'
+	icon_state = "thing"
+	icon_living = "thing"
+	icon_dead =  "shade_dead"
+	maxHealth = 400
+	health = 400
+	melee_damage_lower = 15
+	melee_damage_upper = 20
+	environment_smash = 1
+	universal_speak = 1
+	response_help = "tries to touch the"
+	response_disarm = "attempts to shove the"
+	response_harm = "hits the"
+	attacktext = "enlightens"
+	attack_sound = "enlighten"
+	speak_emote = list("articulates", "speaks")
+	gold_core_spawnable = CHEM_MOB_SPAWN_INVALID
+	atmos_requirements = list("min_oxy" = 0, "max_oxy" = 0, "min_tox" = 0, "max_tox" = 0, "min_co2" = 0, "max_co2" = 0, "min_n2" = 0, "max_n2" = 0)
+	minbodytemp = 0
+	maxbodytemp = 1500
+	friendly = "stares at"
+	faction = list("thing")
+	deathmessage = "crumbles into dust!"
+	lighting_alpha = LIGHTING_PLANE_ALPHA_MOSTLY_VISIBLE
+	var/phasing = 0 // 0 is not phasing, 1 is phasing.
+	var/armblade = 0 // 0 none, 1 transitioning, 2 has it
+	var/prying = 0 // Opening a door? 0 For no, 1 for yes.
+	var/absorbing = 0 // Absorbing?
+	var/phasing_cooldown = 3
+	var/sound_cooldown = 10 // Cooldown to avoid abuse
+	var/absorb_target = null // Thing to absorb.
+	var/points = 0 // Points to spend in upgrades. Starts with 0.
+	var/absorptions = 0 // For special abilities that require you to work hard for them.
+//Abilities
+	var/cantravel = 1 // Can travel? Defaults to yes. Can be manually turned off and it can be bought again.
+	var/canweaponize = 0 // Can weaponize?
+	var/canconvert = 0 // Can convert those who it absorbs into monsters?
+	var/canomnivision = 0 // Can see everything?
```

Comment:
words should be separated with a _

` can_travel`

booleans should also be written as TRUE or FALSE instead of 1 and 0